lang/en.json

```diff
@@ -146,7 +146,7 @@
                 "description": "Un-ban someone from the server",
                 "slashMemberDescription": "Who do you like to un-ban?",
                 "slashReasonDescription": "Un-ban reason",
-                "ubanSuccess": "**{user}** has been **`UN-BANNED`** from the server.",
+                "unbanSuccess": "**{user}** has been **`UN-BANNED`** from the server.",
```

Comment:
Change the key in other localization files as well.

Reply:
:+1: sorry i was distracted didn't saw other files